test/extended/networking.sh

```diff
@@ -24,7 +24,6 @@ NETWORKING_E2E_MINIMAL="${NETWORKING_E2E_MINIMAL:-}"
 
 DEFAULT_SKIP_LIST=(
   # TODO(marun) This should work with docker >= 1.10
-  "openshift router"
   "\[Feature:Federation\]"
```

Comment:
I checked the router tests over again. They are good to go.
I want the router tests to run multi-node because they test the global reachability of the router over the sdn.